### DISCUSSION TITLE
Put examples in ./examples directory

### DISCUSSION THREAD

**Discussion by peter-scholtens** [moka-rs/moka#263] [Ideas]
Hello,
I find it very handy to put both examples in a directory called `examples`. The [tooling of cargo](https://doc.rust-lang.org/cargo/commands/cargo-run.html?highlight=examples#examples) then automatically can find it if you type e.g.:
```
cargo run --example sync_example
```
Or for the async example:
```
cargo run --features="future" --example async_example
```
Can you do that or accept a pull request of it?

**Comment by tatsuya6502** [comment]
@peter-scholtens — Thank you for the suggestion. I believe it is good to have some examples in the `./examples` directory, and yes, pull requests are always welcome!
